index.html

```diff
@@ -58,152 +58,155 @@
-                  >
-                    <circle cx="4.5" cy="4.5" r="4.5" stroke="none" />
-                    <circle cx="4.5" cy="4.5" r="4" fill="none" />
-                  </g>
-                  <line
-                    x1="2"
-                    y1="2.5"
-                    transform="translate(975.5 419.5)"
-                    fill="none"
-                    stroke-width="1"
+                <g transform="translate(-1714 -820)">
+                  <rect
+                    width="43"
+                    height="44"
+                    rx="14"
+                    transform="translate(1714 820)"
+                    fill="#9769c7"
                   />
+                  <g transform="translate(-70 -6.5)">
+                    <line
+                      x2="11"
+                      transform="translate(1800 849)"
+                      fill="none"
+                      stroke="#fff"
+                      stroke-width="1.5"
+                    />
+                    <line
+                      y2="11"
+                      transform="translate(1805.5 843.5)"
+                      fill="none"
+                      stroke="#fff"
+                      stroke-width="1.5"
+                    />
+                  </g>
                 </g>
               </svg>
-            </button>
-            <input
-              class="c-search-bar__input"
-              type="text"
-              placeholder="Search Task or Project..."
-            />
-          </form>
-          <div class="c-search-bar__container">
-            <p class="c-search-bar__container--text"></p>
-            <button class="c-search-bar__container--reset-button">
-              Reset Search
-            </button>
+            </div>
           </div>
         </div>
-
-        <div class="c-projects">
-          <h1>Projects</h1>
-          <ul class="c-projects__list"></ul>
-        </div>
-
-        <div class="c-add-project">
-          <div class="c-add-project__button">
-            <svg
-              xmlns="http://www.w3.org/2000/svg"
-              width="43"
-              height="44"
-              viewBox="0 0 43 44"
-            >
-              <g transform="translate(-1714 -820)">
-                <rect
-                  width="43"
-                  height="44"
-                  rx="14"
-                  transform="translate(1714 820)"
-                  fill="#9769c7"
-                />
-                <g transform="translate(-70 -6.5)">
-                  <line
-                    x2="11"
-                    transform="translate(1800 849)"
-                    fill="none"
-                    stroke="#fff"
-                    stroke-width="1.5"
+        <div class="c-project__container--list">
+          <div class="c-projects-to-do">
+            <div class="c-projects-to-do__header">
+              <div class="c-projects-to-do__header--text-content"></div>
+              <div class="c-projects-to-do__header--close">
```

Comment:
`--close` may or may not be a modifier - what does it do?

Reply:
close svg